buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -65,8 +65,10 @@ class ClusterFormationTasks {
         project.configurations {
             elasticsearchDistro
         }
+        // TODO why? why? why? why? why? why? why? why? why? why? why? why? why? why? why?
+        String artifactId = ['rpm', 'deb'].contains(distro) ? distro : 'elasticsearch'
```

Comment:
You need to add project subsitutions in the root build.gradle file. They only exist for zip and deb right now.